# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1486955574 21600
#      Sun Feb 12 21:12:54 2017 -0600
# Node ID 7520aab218a02ef8e56feed939c53500b9a1508f
# Parent  751c389404b986be40974955fe8395cb5a0249c4
# Parent  6da3526ccf334dc97cf8b06b508a28898aaeb41d
maint: Merge stable to default.

diff --git a/libgui/src/m-editor/file-editor.cc b/libgui/src/m-editor/file-editor.cc
--- a/libgui/src/m-editor/file-editor.cc
+++ b/libgui/src/m-editor/file-editor.cc
@@ -2023,17 +2023,17 @@ file_editor::add_file_editor_tab (file_e
                                                   const QString&,
                                                   const QString&, int)));
 
   connect (f, SIGNAL (set_focus_editor_signal (QWidget*)),
            this, SLOT (set_focus (QWidget*)));
 
   // Signals from the file_editor non-trivial operations
   connect (this, SIGNAL (fetab_settings_changed (const QSettings *)),
-           f, SLOT (settings_changed (const QSettings *)));
+           f, SLOT (notice_settings (const QSettings *)));
 
   connect (this, SIGNAL (fetab_change_request (const QWidget*)),
            f, SLOT (change_editor_state (const QWidget*)));
 
   connect (this, SIGNAL (fetab_file_name_query (const QWidget*)),
            f, SLOT (file_name_query (const QWidget*)));
 
   connect (this, SIGNAL (fetab_save_file (const QWidget*, const QString&,
